design/ShaderLanguageFAQ.md

```diff
@@ -0,0 +1,158 @@
+# FAQ: WebGPU direction on shader language
+
+## What is the shader language for WebGPU?
+
+WebGPU will use a new shader language “WebGPU Shader Language”,
+or “WGSL” for short. This is a shading language which is designed specifically
+for, and in tandem with, WebGPU.
+
+It is under development, but will have the following traits:
+
+*   It is a text-based language.
+*   It is suitable for humans to read and write.
+*   It is strongly typed.
+*   It directly represents block-structured programming, including
+    sequence, selection (if/else), and repetition (looping).
+*   It can be faithfully and simply converted to and from SPIR-V
+    which only uses features available in WebGPU.
+*   It can be faithfully and simply converted to Metal Shading Language
+    which only uses features available in WebGPU.
+*   It can be faithfully and simply converted to HLSL
+    which only uses features available in WebGPU.
+
+The W3C WebGPU community group agreed to this direction during the
+February 2020 face-to-face meeting in Redmond.
+
+
+## How do you pronounce “WGSL”?
```

Comment:
I know there was some controversy over this section.  I'm happy to keep it or remove it as the group desires.